tree/30_generic_methods/sharedfile_to_node.cf

```diff
@@ -152,7 +152,7 @@ bundle agent sharedfile_to_node(target_uuid, file_id, file_path, ttl)
     _stdlib_path_exists_curl::
       "/opt/rudder/bin/rudder-sign"
         args => "${file_path} 1.1",
-        classes => classes_generic("${canonified_path}_signed"),
+        classes => if_ok("${canonified_path}_signed"),
```

Comment:
class is defined l 126: ${canonified_path}_signed" and => { fileexists("${file_path}.sign"), isnewerthan("${file_path}.sign","${file_path}") };